src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2436,6 +2436,7 @@ private function registerMailerConfiguration(array $config, ContainerBuilder $co
             SendinblueTransportFactory::class => 'mailer.transport_factory.sendinblue',
             SesTransportFactory::class => 'mailer.transport_factory.amazon',
             OhMySmtpTransportFactory::class => 'mailer.transport_factory.ohmysmtp',
+            InfobipTransportFactory::class => 'mailer.transport_factory.infobip',
```

Comment:
can you please sort them alphabetically? And please move OhMySmtpTransportFactory, too, thanks